custom_md_loader/classes/MetadataOpType.cls

```diff
@@ -0,0 +1,8 @@
+/* 
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license. 
+ * For full license text, see LICENSE.txt file in the repo root  or https://opensource.org/licenses/BSD-3-Clause
+ */
+ 
+public enum MetadataOpType { APEXWRAPPER, METADATAAPEX }
```

Comment:
Does this need to be a whole class of its own? Or could it be an enum within the class where it's used? Putting it in the right context makes it clearer what it's for.